packages/compass-aggregations/src/modules/index.ts

```diff
@@ -88,14 +88,18 @@ const rootReducer = combineReducers({
 });
 
 export type RootState = ReturnType<typeof rootReducer>;
+export type ExtraArgs = {
```

Comment:
Might be more useful to export a `type PipelineBuilderThunkDispatch<A extends Action = AnyAction> = ThunkDispatch<RootState, ExtraArgs, A>` instead of this very generic and kinda useless otherwise `ExtraArgs`